macos/FluentUITestApp/TestLinkViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+import FluentUI
+
+class TestLinkViewController: NSViewController {
+	override func loadView() {
+		let linkWithNoHover = Link(title: "Link to Bing", url: NSURL(string: "https://wwww.bing.com")!)
```

Comment:
Here's a fun test... what's an `NSURL` that _isn't_ a webpage that we could put here (like a file or image for instance)

Reply:
nit: for iOS we use any example of nsurl to point to this GitHub website lol

Reply:
Curious why an NSURL that doesn´t point to a website is a case. I think this code treats URLs opaquely?

Reply:
I'm pretty sure the NSURL passed in can just as easily be a path to a file as it can be a path to a webpage, distinguished by setting the sheme to `file` or `https`. I feel that there is not a strong reason to restrict this control to only `NSURL`'s with the `https` scheme, especially given a client can easily override the `target/action`

Reply:
Agreed, but do we actually need to test that case? Is that testing the button? Or is that testing AppKit's openURL?

Reply:
Fair. I guess we don't need to test it. Just the realization that I _could_ do that was kinda cool